processor/pom.xml

```diff
@@ -57,7 +57,11 @@
             <artifactId>mapstruct</artifactId>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
```

Comment:
This should better be marked `optional`. `provided` suggests it's needed at runtime but provided by the deployer/container.